File: trialDiv.py
import time
from math import sqrt, floor

def factor(x):
    factors = []
    for i in range(2, floor(sqrt(x))):
        if x % i == 0:
            #print(str(x) + " / " + str(i) + " = " + str(x / i))
            factors.append(i)
            factors.append(x / i)
    return factors

for i in range(40, 61):
    start = time.time()
    factor(2 ** i)
    end = time.time()
    print("Time to factor 2^" + str(i) + " was: " + str(end - start))

File: trialDiv.jl
using Printf

function factor(x)
	a = []
	for i in 2:floor(sqrt(x))
		if x % i == 0
			#output::String = string(x) + " / " + string(i) + " = " + string(x / i)
			#println(string(x, " / ", i, " = ", x / i))
			append!(a, i)
			append!(a, x / i)
		end
	end
end

for i in 40:60
	@time factor(2 ^ i)
end
